src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -77480,13 +77480,13 @@ static void G(object? o)
 
             comp = CreateCompilation(new[] { source }, options: WithNullableEnable());
             comp.VerifyDiagnostics(
-                // (6,18): warning CS8974: Converting method group 'F' to non-delegate type 'object'. Did you intend to invoke the method?
+                // 0.cs(6,18): warning CS8974: Converting method group 'F' to non-delegate type 'object'. Did you intend to invoke the method?
```

Comment:
> 0.cs [](http://example.com/codeflow?start=18&length=5)

It looks like there are no meaningful changes in this file. I recommend restoring it to its original state.